MAINTENANCE.md

```diff
@@ -6,6 +6,12 @@ making releases, creating packages, etc.
 If you want to make a contribution to the project, see the
 [Contributing Guide](CONTRIBUTING.md) instead.
 
+## Onboarding Access Checklist
+
+- [ ] Added to [python-maintainers](https://github.com/orgs/GenericMappingTools/teams/python-maintainers) team in the [GenericMappingTools](https://github.com/orgs/GenericMappingTools/teams/) organization on GitHub (gives 'maintain' permissions)
+- [ ] Added as moderator on [GMT forum](https://forum.generic-mapping-tools.org) (to see mod-only discussions)
+- [ ] Added as member on the [PyGMT devs Slack channel](https://pygmtdevs.slack.com) (for casual conversations)
+- [ ] Added as maintainer on [PyPI](https://pypi.org/project/pygmt/) and [Test PyPI](https://test.pypi.org/project/pygmt) (needed for handling releases)
```

Comment:
The checklist looks good to me. 

> - [ ] Added as maintainer on [PyPI](https://pypi.org/project/pygmt/) and [Test PyPI](https://test.pypi.org/project/pygmt) (needed for handling releases)

This step is usually unnecessary, but releases are handled by github actions automatically.

Reply:
Maybe list hackmd as optional too.